models/perceiver.py

```diff
@@ -140,7 +140,7 @@ def forward(self, x, context = None, mask = None):
             mask = rearrange(mask, 'b ... -> b (...)')
             max_neg_value = -torch.finfo(sim.dtype).max
             mask = repeat(mask, 'b j -> (b h) () j', h = h)
-            sim.masked_fill_(~mask, max_neg_value)
+            sim.masked_fill_(mask, max_neg_value)  # Fills elements of self tensor with value where mask is True
```

Comment:
Mask has TRUEs at the padding position. FALSEs at image position.

https://github.com/Maxvgrad/DePerceiver/blob/b9048ebe86561594f1472139ec42327f00aba699/util/misc.py#L326